assets/performanceprofile/configs/ocp-tuned-one-shot.service

```diff
@@ -36,7 +36,7 @@ ExecStart=/usr/bin/podman run \
     --volume /run/tuned:/run/tuned:rslave \
     --volume /run/systemd:/run/systemd:rslave \
     --volume /sys:/sys:rslave \
-    --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+    --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
```

Comment:
is this correct?

Reply:
It is, any reason why it should not?

Reply:
didn't notice the command line parameter changed as well. If this changed then we're good